ext/zip/php_zip.c

```diff
@@ -2595,12 +2606,12 @@ PHP_METHOD(ZipArchive, unchangeIndex)
 		RETURN_THROWS();
 	}
 
-	ZIP_FROM_OBJECT(intern, self);
-
 	if (index < 0) {
 		RETURN_FALSE;
 	}
```

Comment:
I guess it might make sense to make those sorts of conditions a warning/value error in the future?

Reply:
Yes I kept "api compatible" for now on purpose.